### PR TITLE
dashboard state icons

### PR DESCRIPTION
**Pull request by flynnduism**
Update for the dashboard to show the state icons

* ports over global state styles from 0.1.0
* corrects image path for the new asset structure
* resolved `ionicons`issue so they work correctly in `dashboard.component`

### CODE REVIEW

**Comment by flynnduism** [comment]
@seanknox I'm stumped with the Icon set - you've successfully included and wired up [Ionicons](http://ionicons.com/cheatsheet.html) in the sidebar component, yet I can't get them working here in the dashboard. 

I don't see any special steps in the sidebar component to embed the icons, they're just globally included in the `styles.scss` so I'd expect the dashboard component to be straightforward. 

```
success     -->    .ion-checkmark-circled
failed      -->    .ion-close-circled
```

Note: there is some HTML + CSS that currently hides the icons, but when I adjust those they still do not appear. Network shows the Icon webfont loads successfully. No errors, just blank result.

Are you perhaps using a subset of the full icon set?

**Comment by flynnduism** [comment]
@seanknox disregard the above, I figured it out. Different semantics between ionicons v2 and v3.

File: src/_global_states.scss [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  img {
+    font-size: 1.5rem;
+    float: left;
+    margin: -0.3rem 1rem 0 -0.25rem;
+    max-width: 1.5rem;
+  }
+
+  &.build-success,
+  .success {
+    Color: $success;
```
Comment (seanknox) [inline src/_global_states.scss]: should be `color:` (lowercase)

File: src/app/dashboard/dashboard.component.html [hunk 2]
```diff
@@ -76,7 +72,7 @@ <h4>Project</h4>
                         <div class="status">
                           <div class="success ng-hide" ng-show="projectsbuild.lastBuild.worker.status == 'Succeeded'"><i class="icon ion-checkmark-circled status-success"></i></div>
                           <div class="failed" ng-show="projectsbuild.lastBuild.worker.status == 'Failed'"><i class="icon ion-close-circled status-fail"></i></div>
-                          <div class="active ng-hide" ng-show="projectsbuild.lastBuild.worker.status == 'Active'"><img src="assets/images/rolling.svg" class="status-active"></div>
+                          <div class="active ng-hide" ng-show="projectsbuild.lastBuild.worker.status == 'Active'"><img src="../assets/images/rolling.svg" class="status-active"></div>
```
Comment (seanknox) [inline src/app/dashboard/dashboard.component.html]: Awesome to see the active spinners! Anything under the assets directory i is available from an absolute path when fetching from Angular, so this should be`/assets/images/rolling.svg`.

**Pull request merged**